cli/commands/root.go

```diff
@@ -23,6 +23,11 @@ const hasuraASCIIText = `
 
 `
 
+var (
+	//IsUpdateCommandEnabled is an ldflag used to disable the automatic updates
+	IsUpdateCommandEnabled string
```

Comment:
Can we initialize this value as true so that the `build` step in `Makefile` remains the same ?

Reply:
Removed change on Makefile and initialized this variable